Sources/ApiVideoPlayer/ApiVideoPlayerController.swift

```diff
@@ -15,6 +15,7 @@ public class ApiVideoPlayerController: NSObject {
     private let taskExecutor: TasksExecutorProtocol.Type
     private let multicastDelegate = ApiVideoPlayerControllerMulticastDelegate()
     private var playerItemFactory: ApiVideoPlayerItemFactory?
+    private var storedVideoSpeed: Float?
```

Comment:
As it is only use for iOS >= 16 and macOS 13, `storedVideoSpeed` could be under a `#if`

Reply:
Could not find a way to do it